members/TXL000195.yaml

```diff
@@ -53,6 +53,12 @@ contact_form:
           selector: form#memberForm textarea#message
           value: $MESSAGE
           required: true
+    - javascript:
+        - name: subject truncation
+          selectors: [ "form#memberForm input#subject"]
+          values: [ "$SUBJECT" ]
```

Comment:
You can leave out `values` entirely, since there is no reference to it in the command

Reply:
updated